script.js

```diff
@@ -47,3 +47,199 @@ navLink3.addEventListener('click', () => {
   homePage.style.filter = 'blur(0)';
   logo.style.filter = 'blur(0)';
 });
+
+// popups section
+
+const projectOneCard = {
+  name: 'Tonic',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+  featuredImage: 'src="./images/project1.jpg" alt="project1-snapshot"',
+  technologies: ['html', 'css', 'javascript'],
+  liveLink: 'https://aquinasmoriasi.github.io/Personal-portfolio/',
+  sourceLink: 'https://github.com/Aquinasmoriasi/Personal-portfolio',
+};
+
+const projectOneCardDt = {
+  name: 'Tonic',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. ummy text of the printing and typesetting release of Lorem Ipsum has been the industry\'s  standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  featuredImage: 'src="./images/pj1-alt.jpg" alt="project4-snapshot"',
+  technologies: ['html', 'css', 'javascript', 'github', 'Ruby', 'Bootstrap'],
+  liveLink: 'https://aquinasmoriasi.github.io/Personal-portfolio/',
+  sourceLink: 'https://github.com/Aquinasmoriasi/Personal-portfolio',
+};
+
+const pj1 = `
+<h2 id="pj1-h2">${projectOneCard.name}</h2>
+  <span class="close-btn">&times;</span>
+<div class="highlights">
+    <span class="canopy4">CANOPY</span>
+       <span>
+       <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+       <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+       </svg>
+       </span>
+       <span class="not-canopy4">Back End Dev</span>
+       <span class="not-canopy-alt">Full Stack Dev</span>
+       <span>
+       <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+       <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+       </svg>
+       </span>
+       <span class="not-canopy4">2015</span>
+    </div>
```

Comment:
- [ ] As suggested by the previous reviewer and following the project requirements, your modal popup data is populated with the data from the projects array. This means that the `popup` will contain data from its specific card eg: `title`, `image`, `links`, etc. In your case, all your cards have the same popup with the same data. Kindly, change it and ensure that data is populated dynamically because this will help you add your projects in the future. You will just need to add it in one place. This will be cool no 😊 
**Project requirements**

![image](https://user-images.githubusercontent.com/80612925/175435947-2e2120bd-b680-4320-a556-492c8b053976.png)

- [ ] OPTIONAL: It would be nice if you give an absolute or fixed position to your popup window because it disappears on page scroll. It's a good idea if you also fixed the issues as mentioned by the reviewer.